bindings/go/oci/internal/lister/lister.go

```diff
@@ -114,12 +114,12 @@ func (lister *Lister) List(ctx context.Context, opts Options) ([]string, error)
 	if err != nil {
 		return nil, err
 	}
-	log.Base().Log(ctx, slog.LevelDebug, "listed version candidates", slog.Int("count", len(candidates)))
+	log.Base(ctx).Log(ctx, slog.LevelDebug, "listed version candidates", slog.Int("count", len(candidates)))
```

Comment:
that looks completely off. youre passing the context twice even though the log interface already has a context with it. whats the purpose of the context then?

Reply:
huh, where am I passing it twice?

Reply:
See for example https://github.com/veqryn/slog-context to see how this can be used

Reply:
Ah interesting. I take a look at that package.